src/controllers/TokenController.ts

```diff
@@ -79,20 +80,19 @@ export class TokenController {
     }
 
     public readOneToken(req: Request, res: Response) {
-        if (!req.params || !req.params.address) {
+        if (!req.params || !req.params.tokenWalletAddress) {
             sendJSONresponse(res, 404, { "message": "No address in request" });
             return;
         }
-
         // validate wallet address
-        req.checkParams("address", "wallet address must be alphanumeric").isAlphanumeric();
+        req.checkParams("tokenWalletAddress", "wallet address must be alphanumeric").isAlphanumeric();
```

Comment:
Why is this changed to `tokenWalletAddress`?

Reply:
match url routing

Reply:
What does it mean? `api.trustwalletapp.com/tokens?address=0xe47494379c1d48ee73454c251a6395fdd4f9eb43` this is current endpoint that's being used to retrieve tokens

Reply:
I’m not sure, it’s old logic, I just fix the wrong premeter name.

Reply:
can you revert that name change? I think it will break current logic